### ISSUE TITLE
newmoonnumber dates are different in different forecast files

### ISSUE THREAD

**Issue by emchristensen** [weecology/portalPredictions#110]
So, this seems to be a result of lag time between collecting data and getting it into the repo.  For each csv forecast file in the predictions folder, it makes the moonnumber of the first month of its prediction the next unused moonnumber.  This means that in the file 2017-08-19Allforecasts.csv the first prediction month is August 2017, and is associated with newmoonnumber 496.  However in the file 2017-09-18Allforecasts.csv the first prediction month is now Sept 2017 but is still associated with newmoonnumber 496 because it hasn't gotten any new data between those two predictions (the last data in the repo currently is moon 495).

This could become a problem if we're using all these files to, say, compare how good their predictions of newmoon 496 were.  It's kind of a problem for me, trying to plot the 495 data with the most recent forecast before that data was collected, but I can jerryrig something.

Basically this shouldn't be happening.  We made the newmoonnumber column so we wouldn't have to worry about skipped censuses.  I'm not sure how the prediction files are being made, but it seems like it's probably looking at the moon_dates.csv in the Rodents repo, when it really needs to see the dates of future new moons.

**Comment by sdtaylor** [comment]
Yep, I see what you saying. I haven't run into this  yet because I don't actually use the `forcastmonth` and `forecastyear` columns anywhere. If I want to associate a date with a newmoonnumber then I've been pulling it from the new_moons file.

**Comment by sdtaylor** [comment]
So this is an issue because the forecasted `newmoonnumber` is initiated from the most recent data point, while the forecasted `forecastmonth` and `forecastyear` are initiated from the current date. So if a month of data is missed, then `forecastmonth` and `forecastyear` are incremented +1, while `newmoonnumber` is still at the previous months number.  

We feel all the forecasts are still valid, but some of the 'forecastmonth' and 'forecastyear' columns are probably off. The solution would be to incorporate those values from the newmoon file, instead of just adding 1:11 months to the current date. 


I think we can just change these 2 lines in `PortalForecasts.R` and `PortalHindcasts.R`  
https://github.com/weecology/portalPredictions/blob/2b29603172f1847e89abf6bad275a2faa78b1ca0/PortalForecasts.R#L21
